amethyst_renderer/Cargo.toml

```diff
@@ -25,19 +25,24 @@ opengl = ["gfx_device_gl", "gfx_window_glutin", "glutin"]
 vulkan = ["gfx_device_vulkan", "gfx_window_vulkan"]
 
 [dependencies]
-cgmath = { version = "0.14", features = ["eders", "use_simd"] }
+cgmath = { version = "0.14", features = ["eders"] }
+derivative = "1.0"
 fnv = "1.0"
-gfx = { version = "0.15", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
-gfx_core = { version = "0.7", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
+#gfx = { version = "0.15", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
+gfx = { version = "0.15", features = ["cgmath-types", "serialize"], path = "/home/ekalderon/gfx/src/render/" }
+#gfx_core = { version = "0.7", features = ["cgmath-types", "serialize"], git = "https://github.com/gfx-rs/gfx" }
+gfx_core = { version = "0.7", features = ["cgmath-types", "serialize"], path = "/home/ekalderon/gfx/src/core/" }
```

Comment:
@kvark There were a handful of types that I needed but weren't re-exported in the `gfx` crate, so I had to pull in `gfx_core` directly.